src/ngx_http_lua_util.c

```diff
@@ -1437,9 +1437,6 @@ ngx_http_lua_run_thread(lua_State *L, ngx_http_request_t *r,
 
             ctx->cur_co_ctx = next_coctx;
 
-            ngx_log_error(NGX_LOG_ERR, r->connection->log, 0,
-                          "lua coroutine: %s: %s\n%s", err, msg, trace);
```

Comment:
@bungle `pcall` is a more explicit `try/catch` in the Lua world while many Lua programmers may overlook the error handling in `coroutine.resume`.